motionplan/tpSpaceRRT_test.go

```diff
@@ -430,17 +461,17 @@ func TestPtgCheckPlan(t *testing.T) {
 		test.That(t, err, test.ShouldBeNil)
 
 		pathPose := remainingPlan.Path()[0][ackermanFrame.Name()].Pose()
-		startPose := spatialmath.NewPose(r3.Vector{0, 1000, 0}, pathPose.Orientation())
+		startPose := spatialmath.NewPose(r3.Vector{1000, 0, 0}, pathPose.Orientation())
```

Comment:
I would like reviewers to double check that it is indeed valid to have switched Y value with X.

Reply:
Why are we swapping?